service_management/app/models/analysis/SampleStatuses.scala

```diff
@@ -33,14 +35,18 @@ object SampleStatuses {
         case JsSuccess(Destroyed.key, _)    => JsSuccess(Destroyed)
         case JsSuccess(Contaminated.key, _) => JsSuccess(Contaminated)
         case JsSuccess(Prepared.key, _)     => JsSuccess(Prepared)
+        case JsSuccess(Discarded.key, _)    => JsSuccess(Discarded)
+        case JsSuccess(Dehigdrated.key, _)  => JsSuccess(Dehigdrated)
         case JsSuccess(bad, p)              => JsError(p, s"Unknown sample status code $bad")
         case err: JsError                   => err
       }
     }
 
     implicit val writes: Writes[SampleStatus] = Writes(ss => JsNumber(ss.key))
   }
-
+  /*
+: Ok, Oppbrukt, Kontaminert, Preparert, Kassert og Uttørket
```

Comment:
I would add this as part of the comment for the `SampleStatuses` object, I think.